CODE/Tests/PennyBOTTests/GatewayProcessingTests.swift

```diff
@@ -194,6 +206,41 @@ class GatewayProcessingTests: XCTestCase {
             XCTAssertEqual(canRespond, true)
         }
     }
+    
+    func testAutoPings() async throws {
+        let event = EventKey.autoPingsTrigger
+        await manager.send(key: event)
```

Comment:
sends an event, then checks to make sure DM messages have been sent to people who had registered to get pinged for words